user-service/src/main/java/com/devspark/userservice/controller/UserInfoController.java

```diff
@@ -26,4 +26,9 @@ public ResponseEntity<GetRecomUserListVO> getRecommendUserList(@RequestBody @Val
         return new ResponseEntity<>(getRecomUserListVO, HttpStatus.OK);
     }
 
+    @GetMapping("/my-profile")
+    public ResponseEntity<List<UserInfoEntity>> getAllUsers() {
+        List<UserInfoEntity> users = userInfoService.getAllUserInfo();
```

Comment:
`UserInfoEntity`  should not access and directly return database entity to Frontend since there are some user informations that we don't want to return in response. or use JsonIgnore annotation.